telegram/bot.py

```diff
@@ -3080,16 +3080,17 @@ def set_webhook(
         api_kwargs: JSONDict = None,
         ip_address: str = None,
         drop_pending_updates: bool = None,
+        secret_token: str = None,
     ) -> bool:
         """
         Use this method to specify a url and receive incoming updates via an outgoing webhook.
         Whenever there is an update for the bot, Telegram will send an HTTPS POST request to the
         specified url, containing a JSON-serialized Update. In case of an unsuccessful request,
         Telegram will give up after a reasonable amount of attempts.
 
-        If you'd like to make sure that the Webhook request comes from Telegram, Telegram
-        recommends using a secret path in the URL, e.g. https://www.example.com/<token>. Since
-        nobody else knows your bot's token, you can be pretty sure it's us.
+        If you'd like to make sure that the Webhook was set by you, you can specify secret data in
+        the parameter :paramref:`secret_token`. If specified, the request will contain a header
```

Comment:
```suggestion
        the parameter ``secret_token``. If specified, the request will contain a header
```
no parameter referencing available in v13, same everywhere below